packages/terra-core-docs/src/terra-dev-site/doc/toolbar/About.1.doc.mdx

```diff
@@ -1,5 +1,8 @@
 import { Badge } from 'terra-toolbar/package.json?dev-site-package';
 import DefaultToolbar from './example/DefaultToolbar?dev-site-example';
+import AriaLabelToolbar from './example/AriaLabelToolbar?dev-site-example';
+import AriaLabelledByToolbar from './example/AriaLabelledByToolbar?dev-site-example';
+import AriaControlsToolbar from './example/AriaControlsToolbar?dev-site-example';
```

Comment:
Great effort in adding the 3 new ones here, but having now 4 examples (default, option a, option b, option c...) that only show one part/prop at a time is probably too many for the docsite page and may be confusing to consumers, since we are trying to shift away from showing fragments of each al-a-carte option without context, now towards only showing either the single best composition or multiple full variation examples, that each equally follow best practices so that any would be acceptable if seen in live projects.

Since you have these three ad-hoc and individual tests exactly the same already also as [test files](https://github.com/cerner/terra-core/tree/cd055714965430fb0530d05315a699ba7613b73c/packages/terra-core-docs/src/terra-dev-site/test/toolbar) (these fit better as tests how they are done currently anyway), let go ahead and remove them, and update the "DefaultToolbar" example with fragments of these approaches so we just have one single docsite example that shows the new props in use:
- since we don't have a great stylized visible label UX design that we recommend to go with toolbars, for this docsite example for now, lets focus on using the `ariaLabel` prop, rather than ariaLabelledBy, with label value of "Documentation Tools", 
- plus add the new `ariaControls` and the div+placeholder sample content area below, the div might also help to add an additional aria-label: 
```
      <div id="test-content-id" aria-label="Add Progress Note Documentation">
        <Placeholder title="Sample Content Area for Toolbar" />
      </div>
``` 

\
\
Also - I have added in a separate comment below, we can provide additional content to the docsite example wrapper component (title and description) to give consumers more context.